search/ping.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, Response, request
+
+
+def get_ping_blueprint() -> Blueprint:
+    blueprint = Blueprint('ping', __name__)
+
+    @blueprint.route('/ping', methods=['GET'])
+    def ping() -> Response:  # pylint: disable=unused-variable
```

Comment:
out of curiosity what's the unused variable here?

Reply:
Pylint thinks that the function declaration is a variable and that it's not being used. It appears to be an known issue with Pylint and Flask decorators.

Reply:
Related: https://github.com/elifesciences/profiles/pull/14#discussion_r138336222